src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -71,15 +79,22 @@ public void visitToken(DetailAST ast) {
         final DetailAST defaultGroupAST = ast.getParent();
         //default keywords used in annotations too - not what we're
         //interested in
+        if (ast.getPreviousSibling() != null) {
+            TOKEN_TYPE.add(ast.getPreviousSibling().getType());
```

Comment:
> i did not get what u suggested

You had a list that you keep adding more and more to. You never removed stuff or erased the list, even starting processing a new file. Logic dictates that you will run out of memory if this keeps happening enough. You would also keep using values from old files and this would lead to unexpected results.

> now i am using set within method so issue will not occur.

See new point raised now.